examples/testdata/commands/run/run_test.go

```diff
@@ -16,3 +16,17 @@ func TestRun(t *testing.T) {
 	_, err = td.Run("test1")
 	assert.NoError(t, err)
 }
+
+func TestRunCommand(t *testing.T) {
+	td := testframework.Open()
+	err := td.SetDevboxJson("devbox.json")
+	assert.NoError(t, err)
+
+	_, err = td.Run("ls > test.txt")
```

Comment:
is this doing whatever the current functionality is to do what `devbox shell -- <command>` used to do?

Reply:
@savil basically yes. @ipince can give more context on the difference